src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/IverksettingRepository.kt

```diff
@@ -73,6 +73,17 @@ class IverksettingRepository(val namedParameterJdbcTemplate: NamedParameterJdbcT
                ?: error("Finner ikke iverksett med behandlingId=${behandlingId}")
     }
 
+    //Brukt i patch
+    fun hentAlleMigrerte(): List<UUID> {
+        val alleMigrerteSql = "select data->'behandling'->'behandlingId' as behandling_id from iverksett where data->'behandling'->>'behandlingÅrsak' = 'MIGRERING'"
```

Comment:
Vi har behandling på iverksett-tabellen (til info, må ikke endres)

Smaksak, jeg la querys direkt i patchservicen :) 

```suggestion
        val alleMigrerteSql = "select behandling_id from iverksett where data->'behandling'->>'behandlingÅrsak' = 'MIGRERING'"
```